src/components/TodosLogic.js

```diff
@@ -0,0 +1,77 @@
+/* eslint no-param-reassign: "error" */
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import InputTodo from './InputTodo';
+import TodosList from './TodoList';
+
+const TodosLogic = () => {
+  const [todos, setTodos] = useState([
+    {
+      id: uuidv4(),
+      title: 'Setup development environment',
+      completed: true,
+    },
+    {
+      id: uuidv4(),
+      title: 'Develop website and add content',
+      completed: false,
+    },
+    {
+      id: uuidv4(),
+      title: 'Deploy to live server',
+      completed: false,
+    },
+  ]);
```

Comment:
Please remove the hard coded task in the array, It will makes your codes to look more professional. All tasks should de added from the form.  Please use local storage to store the task so that the tasks will not disappear as soon as the page is refreshed.